src/styles/app.scss

```diff
@@ -57,7 +57,7 @@ $govuk-font-family-tabular: 'helvetica neue',Arial, Helvetica, sans-serif !defau
   padding: 0 govuk-spacing(1);
   overflow-x: auto;
   color: govuk-colour("pink");
-  background-color: govuk-colour("grey-4");
+  background-color: govuk-colour("light-grey");
```

Comment:
This fails the WCAG 2.1 contrast guidelines (https://webaim.org/resources/contrastchecker/).
(The old colour did too, mind)